math/flexiblas/Portfile

```diff
@@ -51,19 +51,26 @@ configure.args-append \
                     -DBLAS_AUTO_DETECT=ON \
                     -DBUILD_SHARED_LIBS=ON \
                     -DDEV=OFF \
+                    -DEXAMPLES=OFF \
                     -DLAPACK=ON \
                     -DLINK_OPENMP=ON \
-                    -DPythonInterp:FILEPATH=${prefix}/bin/python${py_ver} \
-                    -DTESTS=ON \
-                    -DEXAMPLES=OFF \
-                    -DMKL_BUILDER=OFF
+                    -DMKL_BUILDER=OFF \
+                    -DPythonInterp=${prefix}/bin/python${py_ver} \
+                    -DTESTS=ON
 
 if {[string match *clang* ${configure.compiler}]} {
     configure.ldflags-append \
                     -L${prefix}/lib/libomp \
                     -lomp
 }
 
+# Do not depend on atlas by default, since it presently does not build on Catalina and newer,
+# while on older systems it requires building multiple extra gcc/libgcc versions.
+variant atlas description "Build with Atlas support" {
+    depends_lib-append \
+                    port:atlas
```

Comment:
Are you sure that you shouldn't disable it via configure option and change off to on when variant is enabled?

Reply:
@catap It finds whatever BLAS is installed, perhaps no point to specifically disable Atlas if it happens to be present (it does build against it; why I do not make it a direct dependency is because it is broken on new systems).

Reply:
@barracuda156 my point: if user has installed some BLAS as side effect, it picks it on build.

After that user cleanup tree from useless prots, and it may broke this port because it is linked against removed BLAS :)